submissions/xT0PERx/js-dom/code.js

```diff
@@ -0,0 +1,65 @@
+  },
+];
+
+document.querySelector(".menu-list").addEventListener("click", function (e) {
+  if (e.target && e.target.nodeName === "LI") {
+    switch (e.target.className) {
+      case "menu-item-1":
+        showContent("menu-item-1");
+        break;
+      case "menu-item-2":
+        showContent("menu-item-2");
+        break;
+      case "menu-item-3":
+        showContent("menu-item-3");
+        break;
+      case "menu-item-4":
+        showContent("menu-item-4");
+        break;
+      case "menu-item-5":
+        showContent("menu-item-5");
+        break;
+    }
+  }
+});
+
+function showContent(nameClass) {
+  const element = document.querySelector("#content");
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+  const newDiv = document.createElement("div");
+  element.className = "content content-style";
+  for (let index in contentMenu) {
+    if (contentMenu[index].nameClassFromArr === nameClass) {
+      newDiv.innerHTML = contentMenu[index].contentGoods;
+    }
```

Comment:
Try not to use `for...in` for Arrays. Use `for` or `forEach` instead of this (they have better performance). While using `foreEach` you has access to actual element and you don't need to do things like that `contentMenu[index]`.

Reply:
fixed